projects/ion/src/lib/step/step.component.html

```diff
@@ -0,0 +1,53 @@
+<section class="steps" data-testid="ion-step">
+  <div
+    class="step-container"
+    [class.clickable]="
+      clickable && step.status !== 'selected' && step.status !== 'error'
+    "
+    *ngFor="let step of steps; let index = index"
+    (click)="goesTo(step.index)"
+  >
+    <div [class]="disabled ? 'step disabled' : 'step theme-' + theme">
+      <div
+        class="cross"
+        [class.bolded]="this.beforeStepIsChecked(index)"
+        *ngIf="crossVisibility('initial', step)"
+      ></div>
+
+      <div
+        [class]="'step-circle ' + step.status"
+        [attr.data-testid]="'step-' + step.index + '-' + step.status"
+      >
+        <span *ngIf="step.status !== 'checked'">
```

Comment:
Here you can put a NgSwitch or a template in the else to improve readability and organization

[angular documentation talking about NgSwitch](https://angular.io/api/common/NgSwitch)